volatility3/cli/volshell/__init__.py

```diff
@@ -167,6 +167,22 @@ def run(self):
             "-m", "--mac", default=False, action="store_true", help="Run a Mac volshell"
         )
 
+        group = parser.add_mutually_exclusive_group()   
```

Comment:
Same again for volshell.  Having to make the same changes twice should be the alarm bell that some duplication is going on and to check if it's necessary...  5:)